dv_signature_offload_api_example/sig_example.c

```diff
@@ -650,6 +672,43 @@ void set_sig_domain_t10dif(struct mlx5dv_sig_block_domain *domain, void *sig)
 	domain->block_size = (config.block_size == 512) ? MLX5DV_SIG_BLOCK_SIZE_512 : MLX5DV_SIG_BLOCK_SIZE_4096;
 }
 
+void set_sig_domain_t10dif_type2(struct mlx5dv_sig_block_domain *domain, void *sig)
```

Comment:
If type 2 is exactly the same as type 1, we can use the same function. Maybe rename it to type1_2